tests/routes/test_workspaces.py

```
@@ -259,3 +259,11 @@ def test_search_workspace_nonexistent(client: TestClient):
         f"/v2/workspaces/{nonexistent_workspace_id}/search", json="test query"
     )
     assert response.status_code == 200
+
+
+def test_get_or_create_workspace_no_name_no_jwt_workspace(client: TestClient):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_get_or_create_workspace_no_name_no_jwt_workspace`

Add return type annotation: `None`

(ANN201)